addons/website_slides/controllers/main.py

```diff
@@ -347,14 +347,16 @@ def _get_top3_users(self):
         '/slides/<model("slide.channel"):channel>/page/<int:page>',
         '/slides/<model("slide.channel"):channel>/tag/<model("slide.tag"):tag>',
         '/slides/<model("slide.channel"):channel>/tag/<model("slide.tag"):tag>/page/<int:page>',
-        '/slides/<model("slide.channel"):channel>/category/<model("slide.slide"):category>',
-        '/slides/<model("slide.channel"):channel>/category/<model("slide.slide"):category>/page/<int:page>'
+        '/slides/<model("slide.channel"):channel>/category/<int:category_id>',
+        '/slides/<model("slide.channel"):channel>/category/<int:category_id>/page/<int:page>',
     ], type='http', auth="public", website=True, sitemap=sitemap_slide)
-    def channel(self, channel, category=None, tag=None, page=1, slide_type=None, uncategorized=False, sorting=None, search=None, **kw):
+    def channel(self, channel, category_id=0, tag=None, page=1, slide_type=None, sorting=None, search=None, **kw):
         if not channel.can_access_from_current_website():
```

Comment:
Add a small docstring for the uncategorized part.

Something like:

""" Will return all necessary data to display the requested slide_channel.
If we receive category_id = -1, it means we need to display uncategorized slides. """